<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedefineDlgDecl</class>
 <widget class="QWidget" name="RedefineDlgDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Redefine Transaction Type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Invalid Transaction Type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_info">
     <property name="text">
      <string>The transaction below has an unrecognised type/action.
Please select an appropriate entry.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_actionCol">
     <property name="text">
      <string comment="the number of the column containing error">Type/action column</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>2</number>
     </property>
     <row/>
     <row/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select Transaction Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="kcombobox_Actions">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select new investment type</string>
       </property>
       <item>
        <property name="text">
         <string comment="description of investment activity" extracomment="don't translate part after '-'">Buy Shares</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="description of investment activity" extracomment="don't translate part after '-'">Sell Shares</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="description of investment activity" extracomment="don't translate part after '-'">Dividend</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="description of investment activity" extracomment="don't translate part after '-'">Reinvest Dividend</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="description of investment activity" extracomment="don't translate part after '-'">Add Shares</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="description of investment activity" extracomment="don't translate part after '-'">Remove Shares</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interest</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
